_data/toc.yml

```diff
@@ -15,6 +15,8 @@ Advanced:
   - docs/persistence.md
   - docs/best-practices.md
 Reference:
+  - title: GitHub API
```

Comment:
maybe call it `context.github` and put it below the `context` link?

Reply:
Hah, that's how I had it at first, and then couldn't decide.